packages/consumption/test/modules/requests/itemProcessors/createAttribute/CreateAttributeRequestItemProcessor.test.ts

```diff
@@ -33,7 +33,7 @@ describe("CreateAttributeRequestItemProcessor", function () {
     describe("canCreateOutgoingRequestItem", function () {
         test("returns Success when passing an Identity Attribute with owner={{Recipient}}", async function () {
```

Comment:
you didn't update the test names, I think makes the whole thing way more confusing now.